src/main/java/frc/team2412/robot/commands/autonomous/AutoBalanceCommand.java

```diff
@@ -0,0 +1,27 @@
+package frc.team2412.robot.commands.autonomous;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.team2412.robot.subsystems.DrivebaseSubsystem;
+
+public class AutoBalanceCommand extends CommandBase {
+
+	private DrivebaseSubsystem drivebaseSubsystem;
+
+	public AutoBalanceCommand(DrivebaseSubsystem drivebaseSubsystem) {
+		this.drivebaseSubsystem = drivebaseSubsystem;
+		addRequirements(drivebaseSubsystem);
+	}
+
+	@Override
+	public void execute() {
```

Comment:
The robot will need to make sure it will not drive off the charge station (especially the top side) if someone else is on the ramp and we can't balance.

Reply:
Yep, I did notice something interesting in the NavX in that I guess it can return its altitude? But it seems to use barometric pressure to do so, so not sure it would work the best for this sort of thing.

However, the current plan is to only use auto balancing in auto, so someone else being on the charge station shouldn't be an issue. If we do ever plan to try and implement it in teleop, well, let's cross that bridge when we get there.